qiskit_ibm_runtime/qiskit_runtime_service.py

```diff
@@ -561,8 +561,8 @@ def backends(
                     # Get the backends that support OpenPulse
                     QiskitRuntimeService.backends(open_pulse=True)
 
-                For the full list of backend attributes, see the `IBMBackend` class documentation
-                <https://quantum.cloud.ibm.com/docs/api/qiskit/1.4/providers_models>
+                For the full list of backend attributes, see the `IBMBackend class documentation
+                <https://quantum.cloud.ibm.com/docs/api/qiskit-ibm-runtime>`_
```

Comment:
We added the backend model docs back in #2393 but they won't be live yet. I think we can leave the link as this for now.